test/gym_unwrapped_test.py

```diff
@@ -0,0 +1,88 @@
+import gym
+import numpy as np
+from gym import spaces
+
+from supersuit import (
+    clip_actions_v0,
+    clip_reward_v0,
+    color_reduction_v0,
+    delay_observations_v0,
+    dtype_v0,
+    flatten_v0,
+    frame_skip_v0,
+    frame_stack_v1,
+    max_observation_v0,
+    nan_random_v0,
+    nan_zeros_v0,
+    normalize_obs_v0,
+    scale_actions_v0,
+    sticky_actions_v0,
+)
+
+from .dummy_gym_env import DummyEnv
+
+
+def unwrapped_check(env):
+    base_env = env.unwrapped
+
+    # image observations
+    if isinstance(env.observation_space, spaces.Box):
+        if (
+            (env.observation_space.low.shape == 3)
+            and (env.observation_space.low == 0).all()
+            and (len(env.observation_space.shape[2]) == 3)
+            and (env.observation_space.high == 255).all()
+        ):
+            env = max_observation_v0(env, 2)
+            env = color_reduction_v0(env, mode="full")
+            env = normalize_obs_v0(env)
+
+    # box action spaces
+    if isinstance(env.action_space, spaces.Box):
+        env = clip_actions_v0(env)
+        env = scale_actions_v0(env, 0.5)
+
+    # stackable observations
+    if isinstance(env.observation_space, spaces.Box) or isinstance(
+        env.observation_space, spaces.Discrete
+    ):
+        env = frame_stack_v1(env, 2)
+
+    # not discrete and not multibinary observations
+    if not isinstance(env.observation_space, spaces.Discrete) and not isinstance(
+        env.observation_space, spaces.MultiBinary
+    ):
+        env = dtype_v0(env, np.float16)
+        env = flatten_v0(env)
+        env = frame_skip_v0(env, 2)
+
+    # everything else
+    env = clip_reward_v0(env, lower_bound=-1, upper_bound=1)
+    env = delay_observations_v0(env, 2)
+    env = sticky_actions_v0(env, 0.5)
+    env = nan_random_v0(env)
+    env = nan_zeros_v0(env)
+
+    assert env.unwrapped == base_env, f"Failed to unwrap {env}"
```

Comment:
Could you do something along the lines of `env.unwrapped.__class__ == DummyEnv`?